BossMod/ActionQueue/ActionDefinition.cs

```diff
@@ -53,11 +54,13 @@ public sealed record class ActionDefinition(ActionID ID)
     public ActionAspect Aspect; // useful for BLM and BLU
     public int MaxChargesBase; // baseline max-charges when action is unlocked
     public readonly List<(int Charges, int Level, uint UnlockLink)> MaxChargesOverride = []; // trait overrides for max-charges (applied in order)
+    public bool IsRoleAction; // unlocked conditions are different for these
     public float InstantAnimLock = 0.6f; // animation lock if ability is instant-cast
     public float CastAnimLock = 0.1f; // animation lock if ability is non-instant
     public ConditionDelegate? ForbidExecute; // optional condition, if it returns true, action is not executed
     public SmartTargetDelegate? SmartTarget; // optional target transformation for 'smart targeting' feature
     public TransformAngleDelegate? TransformAngle; // optional facing angle transformation
+    public TransformPositionDelegate? TransformPosition; // optional ground-targeted action position transformation
```

Comment:
Ok this one I don't quite understand, what exactly are you trying to achieve?

Reply:
for ground targeting, specifically for actions that can't target enemies - AST Earthly Star, whatever SCH and WHM's ground targeted heals/mits are called (and caster LBs if we ever want to cdplan those)